_schema.graphql

```diff
@@ -1615,7 +1617,9 @@ type ArtworkContextSale implements Node {
   href: String
   name: String
   is_auction: Boolean
-  is_benefit: Boolean
+  is_benefit: Boolean @deprecated(reason: "Favor `isBenefit`")
```

Comment:
<!--
  0 failure: 
  1 warning:  Addition uses `sn...
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 1620;
-->

- :warning: Addition uses `snake_case` instead of `camelCase`: `is_benefit`